tripal/src/Entity/TripalEntity.php

```diff
@@ -154,14 +162,26 @@ public function getTitle() {
   /**
    * Generates a default URL alias for the current entity.
    *
+   * @param string $default_alias
+   *   Either an empty string if default alias is desired,
+   *   or an alias that may optionally contain tokens
+   *
    * @return string
    *   The default entity alias, e.g. "/project/1234"
    */
-  public function getDefaultAlias() {
-    // Generate an alias using the default format set by admins.
+  public function getDefaultAlias(string $default_alias = '') {
```

Comment:
It would be good to add the $cache here as well for consistency -that must have been missed on my part earlier :see_no_evil

Reply:
done